Examples/js/index.html

```diff
@@ -27,14 +27,14 @@
 <script>
 var Module = {
   preRun: [function() {
-    Module.FS_createPreloadedFile('/', 'ORBvoc.txt', 'ORBvoc.txt', true, false);
+    Module.FS_createPreloadedFile('/', 'ORBvoc.bin', 'ORBvoc.bin', true, false);
     Module.FS_createPreloadedFile('/', 'c270.yaml', 'c270.yaml', true, false);
   }],
   _main: function() {opencvIsReady();}
 };
 </script>
-<script async src="opencv.js"></script>
+<script src="opencv_js.js"></script>
+<script>
```

Comment:
I remember we added `cv = cv(Module)` here, then following code doesn't change. Does that work?

Reply:
No. Because of https://github.com/kripken/emscripten/issues/5009 , I remove -s MODULARIZE=1 -s EXPORT_NAME=\"'cv'\" options. So cv is undefined here.

Reply:
Got it. It makes sense.